.github/scripts/addUnblockedComment.js

```diff
@@ -0,0 +1,56 @@
+// @ts-check
+// When a blocking issue is closed:
+// 1. Leaves a comment on all the issues listed as blocked in body,
+
+/** @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments */
+module.exports = async ({ github, context }) => {
+  const { repo, owner } = context.repo;
+  const payload = /** @type {import('@octokit/webhooks-types').IssuesLabeledEvent} */ (context.payload);
+  const { ISSUE_VERIFIERS } = process.env;
+  const issueBody = payload.issue.body;
+  const blockedIssuesRegex = /(?!Blocked issues:\s)(#\d+)/gi;
+
+  if (!issueBody) {
+    console.log("No issue body was found");
+    return;
+  }
+
+  // Get the list of issues blocked by this issue if there are any
+  const blockedIssues = issueBody.match(blockedIssuesRegex);
+
+  // Add a "@" character to notify the user
+  const verifiers = ISSUE_VERIFIERS?.split(",").map((v) => `@${v.trim()}`);
+
+  // If "Blocked issues" line is matched in the body then create a comment on each issue listed
+  if (blockedIssues) {
+    const issueNumbers = blockedIssues.map((number) => number.slice(1));
+
+    for (const issueNumber of issueNumbers) {
+      const issueProps = {
+        repo,
+        owner,
+        issue_number: issueNumber,
```

Comment:
I'm seeing type errors for the `createComment` and `removeLabel` function calls. Are you not seeing this on your end?

> tsserver: Argument of type '{ body: string; repo: string; owner: string; issue_number: string; }' is not assignable to parameter of type 'RequestParameters & { owner: string; repo: string; issue_number: number; body: string; }'.
  Type '{ body: string; repo: string; owner: string; issue_number: string; }' is not assignable to type '{ owner: string; repo: string; issue_number: number; body: string; }'.
    Types of property 'issue_number' are incompatible.
      Type 'string' is not assignable to type 'number'. [2345]
 
 I wonder if I need to fix something with the types setup. Regardless, `issueNumber` needs to be a `number`, not a `string`.

```suggestion
        issue_number: Number(issueNumber),
```